{
  try {
    join();
    return 0;
  }
 catch (  InterruptedException e) {
    throw L.errRTExcept(e);
  }
}
